python/mxnet/ndarray.py

```diff
@@ -517,9 +517,9 @@ def reshape(self, shape):
 
     # pylint: disable= undefined-variable
     def broadcast_to(self, shape):
-        """Broadcasts an array to a new shape.
+        """Broadcasts the input array to a new shape.
```

Comment:
Added one line explanation and pointed out to the mentioned link. Check file broadcast_reduce_op_value.cc